cadc-vos-server-db/src/main/java/ca/nrc/cadc/vos/server/db/NodeDAO.java

```diff
@@ -733,7 +734,7 @@ private void getOwners(Node node, boolean resolve)
             return;
 
         NodeID nid = (NodeID) node.appData;
-        if (nid.owner != null)
+        if (nid.getOwner() != null)
```

Comment:
would prefer to keep the old style (ref field rather than getter) since fields are public now... mostly for fewer changes but also because better class design doesn't have public fields and redundant getters

same on line 2742

please revert these shanges

Reply:
Done